src/ensembl/src/content/app/entity-viewer/gene-view/components/default-transcripts-list/DefaultTranscriptsList.tsx

```diff
@@ -29,6 +29,7 @@ const DefaultTranscriptslist = (props: Props) => {
         </div>
       </div>
       <div className={styles.content}>
+        <StripedBackground {...props} />
         {sortedTranscripts.map((transcript, index) => (
```

Comment:
Sorry, we let the transcript-sorter into dev; but shouldn't the sorting rule be the opposite to the [accepted one](https://github.com/Ensembl/ensembl-client/blob/dev/src/ensembl/src/content/app/entity-viewer/shared/helpers/transcripts-sorter.ts#L14-L20), i.e.:

```
  if (transcriptOneLength > transcriptTwoLength) {
    return -1;
  }

  if (transcriptOneLength < transcriptTwoLength) {
    return 1;
  }
```

to get this:

![image](https://user-images.githubusercontent.com/6834224/78685333-85756d80-78e9-11ea-9384-efde8a776b3a.png)

rather than this:

![image](https://user-images.githubusercontent.com/6834224/78685519-c1a8ce00-78e9-11ea-8973-964292f74263.png)

I seem to recall that that was Andrea's intent.

Reply:
Yes, that's true. I'll fix it in this PR itself.

Reply:
✔️